### PR TITLE
DOCSP-1034a: fixed incorrect image

### PR DESCRIPTION
**Pull request by steveren**

### CODE REVIEW

**Comment by steveren** [comment]
Staged at: https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/steverenaker/DOCSP-1034a/connect/tableau.html#connect-from-tableau-without-authentication-or-tls-ssl

File: source/connect/tableau.txt [hunk 1]
```diff
@@ -31,7 +31,7 @@ listening, and click :guilabel:`Sign In` to connect.
 
 By default, :program:`mongosqld` binds to port ``3307``.
 
-.. figure:: /images/bi-connector/tableau.png
```
Comment (atsansone) [inline source/connect/tableau.txt]: Believe it or not, the image needs to go here: https://github.com/mongodb/docs-assets/tree/bi-connector

**Comment by breinero** [comment]
LGTM

File: source/connect/tableau.txt (hunk 1 above)
Comment (atsansone) [inline source/connect/tableau.txt]: Also, just caught that I messed up the *Server* field. It should probably say `localhost` like the others.

**Comment by kay-kim** [comment]
@steveren / @atsansone  -- the images are kept within the bi-connector repo going forward.  An email about this was sent to the group in January.

**Comment by kay-kim** [comment]
@steveren  -- if you could fix the second image as a separate commit to this branch, I can merge and publish.

**Comment by atsansone** [comment]
@kay-kim : It did not work when I tried it a week ago. @i80and told me to put it in `docs-assets` and it worked. I'll circle back.

**Comment by kay-kim** [comment]
It didn't work a week ago because after the email went out in January, no one actually took that step for their respective repo.  I noticed it while doing a merge on Monday, so I went ahead and made the change to this repo -- well, specifically, the master branch anyhow.

**Comment by i80and** [comment]
Sorry for the wrong instructions, @atsansone. I should have remembered that we were moving away from docs-assets.

**Comment by steveren** [comment]
OK, I believe all the images are now correct. I fixed the other one which still had MySQL in it and updated the one which had 127.0.0.1 instead of localhost.

**Comment by kay-kim** [comment]
merged!

**Pull request closed without merging**